packages/react/src/themes/teams/components/Alert/alertVariables.ts

```diff
@@ -55,12 +56,13 @@ export default (siteVars: SiteVariablesPrepared): AlertVariables => {
     color: siteVars.colors.grey[500],
     fontWeight: siteVars.fontWeightRegular,
     minHeight,
-    padding: `0 0 0 ${pxToRem(16)}`,
+    padding: `0 ${pxToRem(16)}`,
```

Comment:
@codepretty can you please check this one? Is it correct?

Reply:
Looks good to me